style.css

```diff
@@ -1,340 +1,1403 @@
+    transition-property: border-color, background-color;
+    transition-duration: .2s;
+    position: relative;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.form-checkbox__box::before {
+    content: '';
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABkAAAATCAYAAABlcqYFAAAACXBIWXMAACE4AAAhOAFFljFgAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAADoSURBVHgBtdPtDYIwEAbgO4Oi4Q+O5CTqJIygTiAbMIpugol811h7DZLKl1DKmxQIbe6h6QFgMGnOvChjIY04f11Dzl0wGSoaZ4yrI8lYQHMIBkIAcH5omytsazsZ6QMoznqJC5gRAERf3mAmABHum5W1Q8SH3InaFeKwTmAQkC+oaL0rZJEeoLFe7aic3XitdVFMhOLutnyO79jWcdIOynQfvCim7kgXoFhvDpcFgtcDVc+gAch5ukRp4YsFe9DIP6BCdKEhwA8yFhoKNJCh0BigMwSN+Q+MQkaBb54JO5dAKEZgHDCZD1zmI/Su9qWCAAAAAElFTkSuQmCC');
+    background-position: center;
+    background-size: 8px 6px;
+    background-repeat: no-repeat;
+    opacity: 0;
+    transition: opacity .2s ease;
+}
+
+.form-checkbox__text {
+    color: #F5F8FB;
+    font-size: 14px;
+    line-height: 1.4em;
+}
+
+/* form switcher */
+
+.form-switcher {
+    padding: 24px;
+    display: flex;
+    justify-content: space-between;
+    background-color: #1D1C28;
+}
+
+.form-switcher__text {
+    font-size: 14px;
+    line-height: 1.4em;
+    color: #F5F8FB;
+}
+
+.form-switcher__box {
+    display: block;
+    width: 40px;
+    height: 20px;
+    border-radius: 19px;
+    background-color: rgba(229, 236, 251, 0.36);
+    position: relative;
+    cursor: pointer;
+    transition: background-color .2s ease;
+}
+.form-switcher__box::before {
+    content: '';
+    display: block;
+    position: absolute;
+    width: calc(100% + 6px);
+    height: calc(100% + 6px);
+    left: 50%;
+    top: 50%;
+    opacity: 0;
+    transform: translate(-50%, -50%);
+    transition: opacity .2s ease;
+    pointer-events: none;
+    border: 2px solid var(--color-blue);
+    border-radius: 100px;
+}
+
+.form-switcher__box::after {
+    content: '';
+    width: 16px;
+    height: 16px;
+    border-radius: 50%;
+    background-color: #FFFFFF;
+    position: absolute;
+    top: 2px;
+    left: 2px;
+    transition-property: transform, background-color;
+    transition-duration: .2s;
+}
+
+.form-switcher__input:checked + .form-switcher__box {
+    background-color: var(--color-blue);
+}
+.form-switcher__input:checked + .form-switcher__box::after {
+    transform: translateX(calc(100% + 4px));
+}
+.form-switcher__input:checked + .form-switcher__box::before {
+    opacity: 1;
+}
+
+.form-switcher__input:disabled + .form-switcher__box {
+    background-color: rgba(208, 215, 251, 0.17);
+    cursor: default;
+    opacity: .5;
+}
+
+.form-switcher__box:hover {
+    background-color: rgba(212, 217, 242, 0.26);
+}
+.form-switcher__input:checked + .form-switcher__box:hover {
+    background-color: var(--color-blue);
+}
+
+/* form radio */
+
+.form-radio {
+    display: inline-flex;
```

Comment:
Please remove. According to design radio buttons should be in column.

Reply:
done